README.md

```diff
@@ -22,10 +38,32 @@ We are using [Turborepo](https://turborepo.org/) to compile and build our applic
 - 5432: Has a docker container with Postgres running
 - 4000: Will have the node app, after starting it with `npm run start:dev` using default settings
 
-##### New Backend Project
+>#### New Backend Project
```

Comment:
Im not sure we want to keep the block quotes here.

Reply:
was such an idea but if you do not like it, I put it out.